packages/web-client/app/components/action-chin/index.hbs

```diff
@@ -0,0 +1,26 @@
+<Boxel::ActionChin @state={{@state}} @stepNumber={{@stepNumber}} ...attributes>
+    <:default as |a|>
+        {{yield (hash
+          ActionButton=(if @frozen (component a.ActionButton disabled=true) a.ActionButton)
```

Comment:
Would it be functionally equivalent and more succinct to write:

`ActionButton=(component a.ActionButton disabled=@frozen)`

Reply:
Seems like it! Thanks for pointing it out.